### DISCUSSION TITLE
Instagram feed - hidden / removed content

### DISCUSSION THREAD

**Discussion by pi-eaterVZ** [gildas-lormeau/SingleFile#1398] [Q&A] [answered]
To begin with, thanks for youre wonderfull extension ! its great and i use it all the time ! Great work. 

**Is your feature request related to a problem? Please describe.**
Being logged in with an Instagram account, when trying to save an Instagram Feed ( posts )  - some content is being removed / being hidden / .... in the source code. The content is not saved by singlefile.
The content seems to be removed / hidden by Instagram when scrolling down and loading more posts.  When i look in the source code i cant found the removed content, it not simply 'hidden' (?).
The result of the mhtml - generated with single file - is a white area and the last posts that where loaded / in viewport.

**Describe the solution you'd like**
Being able to fully capture the page.

**Describe alternatives you've considered (optional)**
Havent found any other options.

**Additional context (optional)**

**Comment by gildas-lormeau** [comment, marked as answer]
Thank you for the kind words. I confirm I was able to reproduce the issue on https://www.instagram.com/the_daily_mirror/ (a random feed I found on Google) without being logged-in.

The problem can be circumvented by enabling the option `Images > save deferred images > zoom out the page` in SingleFile.

**Comment by pi-eaterVZ** [comment]
Thank you for the feedback / suggestion , i tried changing the settings and it works as expected now. Great !
